Mathlib/Geometry/Manifold/IsManifold/Basic.lean

```diff
@@ -132,7 +132,15 @@ noncomputable section
 
 universe u v w u' v' w'
 
-open Set Filter Function
+namespace PartialEquiv
+
```

Comment:
Can you add a comment explaining why this lemma is added, and why it is placed here (and not in `PartialEquiv.lean`)? Basically, what you said in the PR discussion.